.gitignore

```diff
@@ -128,9 +128,6 @@ src/frontend/i18n/frontend.json
 src/backend/marsha/static/css
 src/backend/marsha/static/js
 
-# Assets
-data/
```

Comment:
You should just ignore the data at the root of the project

```
./data/
```

Reply:
But is there still a reason to gitignore that folder? AFAIK it's not used anymore, is it?

Reply:
Yes, I added this ignore instruction in PR #882 because we are managing statics in the docker image directly and in my tests statics are made in this `data` directory.

Reply:
Oh ok, great, I'll do what you suggested then 👍